data/map.txt

```diff
@@ -22450,7 +22475,7 @@ planet "Ablub's Invention"
 	"required reputation" 25
 
 planet Ada
-	attributes paradise mining
+	attributes paradise shipyard syMinor mining
```

Comment:
I intend to create jobs where you deliver ship components from some syMajor to some syMinor destination.
I.e. some parts are only manufactured in the big yards...